app/(public)/_components/footer.tsx

```
@@ -0,0 +1,158 @@
+
+                <div className="mt-12 flex flex-col items-center justify-between border-t pt-8 md:flex-row">
+                    <p className="text-muted-foreground text-sm">
+                        © 2024 ArchMentor. All rights reserved.
+                    </p>
+                    <div className="mt-4 flex space-x-4 md:mt-0">
+                        <Link
+                            href="#"
+                            className="text-muted-foreground hover:text-primary transition-colors"
+                        >
+                            <span className="sr-only">Twitter</span>
+                            <svg className="h-5 w-5" fill="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)